README.md

```diff
@@ -35,11 +35,11 @@ conda install -c conda-forge scikit-rf
 ## Questions or comments? Ask the Community
 
 [![Mailing List](https://img.shields.io/badge/mail-Mailing%20list-brightgreen)](http://groups.google.com/group/scikit-rf)
-[![Slack](https://img.shields.io/badge/Slack-join%20chat-blueviolet?style=flat&logo=slack)](https://scikit-rf.slack.com)
+[![Slack](https://img.shields.io/badge/Slack-join%20chat-blueviolet?style=flat&logo=slack)](https://join.slack.com/t/scikit-rf/shared_invite/zt-d82b62wg-0bdSJjZVhHBKf6687V80Jg)
 [![Matrix](https://img.shields.io/badge/Matrix-join%20chat-blueviolet?style=flat&logo=matrix)](https://app.element.io/#/room/#scikit-rf:matrix.org)
 
 * [Mailing list](http://groups.google.com/group/scikit-rf): You can suscribe to our mailing list to ask (and answer) questions and to receive scikit-rf announcements.
-* Chat: scikit-rf have channels on both [Slack](https://scikit-rf.slack.com) and [Matrix](https://app.element.io/#/room/#scikit-rf:matrix.org)  
+* Chat: scikit-rf have channels on both [Slack](https://scikit-rf.slack.com) (invitation link is above) and [Matrix](https://app.element.io/#/room/#scikit-rf:matrix.org)  
```

Comment:
Why not directly link to the correct url with the invitation link?

Reply:
yes, that's right.